### PR TITLE
chore: update deps

### PR DESCRIPTION
**Pull request by 47vigen**
update material ui to the latest version. @mui/material@6.0.1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,9 +68,9 @@
     }
   },
   "dependencies": {
-    "@emotion/react": "^11.11.4",
-    "@emotion/styled": "^11.11.5",
-    "@mui/material": "^5.16.0",
+    "@emotion/react": "^11.13.3",
+    "@emotion/styled": "^11.13.0",
+    "@mui/material": "^6.0.1",
```
Comment (viclafouch) [inline package.json:73]: We have to stay compatible with v5

**Comment by ruiaraujo012** [comment]
You should update the peer-deps
```
.
└─┬ mui-one-time-password-input 2.0.3
  └── ✕ unmet peer @mui/material@^5.0.0: found 6.1.1
  ```

File: package.json (hunk 1 above)
Comment (ruiaraujo012) [reply package.json:73]: If the upgrade is not possible without being compatible with v5, a major release of the package should be released to support v6.

| mui-otp-input | Mui |
| --- | --- |
| v2 | v5 |
| v3 | v6 |

**Comment by brandaspt** [comment]
Any update on this?

**Comment by viclafouch** [comment]
Hello ! 

This package is now only v6 compatible : https://github.com/viclafouch/mui-otp-input/releases/tag/v3.0.0

Thanks for your help guys !

**Comment by ruiaraujo012** [comment]
Thanks.

**Pull request closed without merging**